### DISCUSSION TITLE
Notifications, not social feed

### DISCUSSION THREAD

**Discussion by msharov** [community/community#13335] [Other Feature Feedback, Questions, & Ideas]
The feed as it is now is utterly useless; I couldn't care less if somebody I never heard of stars of forks one of my repositories.

A much better use for that space would be to list the events that currently go into the notifications area. Like being notified that an issue was opened on one of my repositories. I don't check them all daily and if a notification is accidentally dismissed by mark all read, then there is no record of it having happened. Keeping issue creation in the feed would remind me to address it and would actually make the feed of some use.

It's the same with the other notification types. They are generally things that I would want to respond to somehow, and that is exactly the sort of thing that belongs in the feed. All the "star", "fork", "follow", etc. belongs in the "for you" feed instead, which I'll just turn off and never look at again. No offence, but your recommendation algorithms suck. If I wanted such things, I'd be on facebook.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
